README.md

```diff
@@ -74,10 +74,10 @@ asdf local deno 0.2.10
   One application of this is a system-wide Shell installation to [`/opt/deno`](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s13.html):
 
   ```sh
-  curl -fsSL https://deno.land/x/install/install.sh | DENO_INSTALL=/opt/deno sh -s v0.2.10
+  curl -fsSL https://deno.land/x/install/install.sh | sudo DENO_INSTALL=/opt/deno sh
```

Comment:
Maybe this directory should be changed to something more common like /usr/local/bin

Reply:
Done :(

Reply:
Removed redundant `bin`.